indico/modules/rb/controllers/backend/admin.py

```diff
@@ -503,3 +504,45 @@ def _process_POST(self, args):
         db.session.flush()
         RHRoomsPermissions._jsonify_user_permissions.clear_cached(session.user)
         return jsonify(id=room.id)
+
+
+_base_schema = {
+    'default': fields.Bool(missing=False),
```

Comment:
are you sure this is correct when used in the `update_schema`? unless you always send a value for `default` (in which case it should be `required`), omitting it will be just like sending `False`.

Reply:
Actually I think I might remove it!